testapi.pm

```diff
@@ -636,6 +673,8 @@ C<$wait_seconds> defaults to 2 seconds.
 
 I<The implementation is distribution specific and not always available.>
 
+TODO: Make this compatible with serial terminals
```

Comment:
better not add 'TODOs' to the code which will stay there forever unless someone wants to really fix it. I suggest to keep "open points" in tickets and incomplete implementation including TODOs in code within your local repo. You can keep branches with experimental content.